modules/dlm-lifecycle-policy/README.md

```diff
@@ -0,0 +1,25 @@
+## Data Lifecycle Manager (DLM) lifecycle policy for managing snapshots
+
+This module creates the policy that manage the creation of EBS snapshots through AWS Data Lifecycle Manager, the policy let you manage the schedule of the snapshot as well as the number of snapshots.
+
+### Example how to use
+
+Define variables
+
+```
+module "ebs-backup-policy" {
+  source = "github.com/fpco/terraform-aws-foundation//modules/dlm-lifecycle-policy"
```

Comment:
This example should source the module from the registry (we need a ticket for making sure all of our examples and docs do that - #268).